packages/apps-v5/src/commands/ps/type.js

```diff
@@ -95,6 +94,10 @@ Types: ${cli.color.yellow(formation.map((f) => f.type).join(', '))}`)
     if (isShowingEcoCostMessage) cli.log('\n$5 (flat monthly fee, shared across all Eco dynos)')
   }
 
+  if (context.args[0] === 'free') {
```

Comment:
Do we need this? The API will give us this error if somebody tries to provision a free dyno.

Reply:
For this instance, no, but for command instances like `ps:resize` with `restrict-free-dynos` enabled, it still runs successfully. 

```
$ heroku ps:resize free -a test-container-app-z
 ›   Warning: heroku update available from 7.65.0 to 7.66.2.
Scaling dynos on ⬢ test-container-app-z... done
=== Dyno Types
type  size  qty  cost/mo
────  ────  ───  ───────
web   Free  1
=== Dyno Totals
type  total
────  ─────
Free  1

Reply:
@zwhitfield3 That's something that needs to be enforced at the API level, so if that's not the case, that's either not completed yet or it's an oversight. Can you please check in with them about the cases where you're seeing free allowed with the flag enabled?